src/components/discovery/LocusSearch.js

```diff
@@ -62,11 +66,32 @@ const LocusSearch = ({assemblyId, addVariantSearchValues, handleLocusChange, set
 
     const handleOnBlur = () => {
         // antd has no "select on tab" option
-        // so check if there's a valid locus when tabbing away
-        handleLocusChange(currentLocus);
+        // so when tabbing away, handle the current contents of the input
+        // input can be one of three cases:
+        // - an autocomplete selection
+        // - position notation
+        // - incorrect
+
+        // incorrect values are passed as null for handling elsewhere
+
+        const isAutoCompleteOption = inputValue.endsWith("_autocomplete_option");
+        const isPositionNotation = inputValue.includes(":");
+
+        if (!(isAutoCompleteOption || isPositionNotation)) {
+            handleLocusChange({chrom: null, start: null, end: null});
+            addVariantSearchValues({chrom: null, start: null, end: null});
+            return;
+        }
+
+        if (isPositionNotation) {
+            const {chrom, start, end} = parsePosition(inputValue);
```

Comment:
parsePosition always returns `{chrom, start, end}` shape, so there's not too much point in deconstructing & then reconstructing the object immediately for handleLocusChange/addVariantSearchValues IMO

Reply:
yeah in this case parsePosition doesn't return an assemblyID though right?

Reply:
yes true

Reply:
Fixed this one, and restructured locus to pull out another one.